create_infotexts.py

```diff
@@ -195,6 +195,8 @@ def main():
         img = CypernImage()
         img.generate_list_of_stripped_keywords(metadata[fotonr]["Nyckelord"])
         img.create_commons_filename(metadata[fotonr])
+        img.add_cat_archaeological_exhibition_based_on_description(metadata[fotonr]["Beskrivning"])
```

Comment:
B) I still think checking for words in the description and adding a one-to-one category is "only one thing". I'll explain my reasoning below but I'm also ok with keeping it as it is.

Essentially the two functions you have are equivalent to a mapping which you are looping over
```python
words_to_cats = {
  "Archaeological_exhibitions": ("utställning", ),
  "Interiors_of_tombs": ("interiör", "grav)
}

for cat, words in words_to_cats.items():
    if all(word in description.lower() for word in words):
        self.content_cats.append(cat)
```
(note I haven't checked that the above code actually runs)